tsl/test/expected/dist_ddl.out

```diff
@@ -7,31 +7,25 @@
 \unset ECHO
 psql:include/remote_exec.sql:5: NOTICE:  schema "test" already exists, skipping
 psql:include/filter_exec.sql:5: NOTICE:  schema "test" already exists, skipping
--- Cleanup from other potential tests that created these databases
-SET client_min_messages TO ERROR;
-DROP DATABASE IF EXISTS data_node_1;
```

Comment:
Aren't we still need those lines?

Reply:
Nope.

Reply:
To elaborate. The databases should not exist prior to this run. The only test that actually creates these databases is this one, and it drops the databases at the end. If the databases already exists, there is an error in the test or in the environment that execute the tests, so we do not need to drop the databases before.

Reply:
But if this test will fail in the middle of execution and run agaiin, base directory with those databases will be created from scratch, right?

Reply:
AFAICT, yes. `pg_regress` will create a new instances each time when you run it locally.

Just tested by introducing a syntax error in the `dist_ddl` and executing that test only. Modifying the file to restore the original, and it passes.